code/modules/mob/living/simple_animal/bot/medbot.dm

```diff
@@ -534,7 +534,7 @@
 			C.visible_message(span_danger("[src] is trying to tend the wounds of [patient]!"), \
 				span_userdanger("[src] is trying to tend your wounds!"))
 
-			if(do_mob(src, patient, 20)) //Slightly faster than default tend wounds, but does less HPS
+			if(do_mob(src, patient, 200)) //SKYRAT EDIT: Increased time as tradeoff for automated healing.
```

Comment:
document the old value (20) in the comment

Reply:
Here you go.
```suggestion
			if(do_mob(src, patient, 20 SECONDS)) //SKYRAT EDIT: Increased time as tradeoff for automated healing. ORIGINAL: if(do_mob(src, patient, 20))
```

Reply:
Thank you both! Should now be in line with this.